packages/core-ethereum/package.json

```diff
@@ -12,7 +12,7 @@
     "test": "mocha --parallel",
     "dev": "yarn clean && tsc -w",
     "build": "yarn clean && tsc",
-    "prepack": "../../scripts/remove-tests.sh",
+    "prepack": "../../scripts/remove-tests.sh packages/core-ethereum",
```

Comment:
How about you just provide the package name and let the script create the path?

Reply:
changed